spec/models/note_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Note, type: :model do
```

Comment:
Worth deleting this or filling it out?

Reply:
It 'should' get filled out really, but I can delete for MVP. what approach have you guys been doing yesterday?  Did you talk tests? I didn't see anything in the main chat.

Reply:
Yes, we've added some basic tests, check out the other model specs.

Reply:
I've been writing the specs.  I did start on doing a proper integration test for a controller, but was concerned by amount of work vs payoff it would be given the current changes in designs.

Agree we need to have a chat about test strategy soon though

Reply:
Agree. My ad-hoc strategy so far - write quick unit-tests only, skip integration tests for now. The unit tests just guard against small overlooks like typos, breaking associations and stuff. I don't think we need intricate tests for the MVP.